src/Core/DesignTokens/Services/ThemeConstant.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Microsoft.Fast.Components.FluentUI.DesignTokens;
+
+internal sealed class ThemeConstant
```

Comment:
These constants are used only one time. It's better to define in the associated class.